Fix some more minor misspellings to smooth it out

jeffadev opened the pull request:
Hey, back again.

This time I went through the entire locale YML file to spot any off-puttings. This is what I came up with so far. Hopefully for the English speakers the messages will be much smoother.

Thanks!

jeffadev committed:
Some more changes to English locale.

jeffadev committed:
Fix some colon spacing, and s

Zedd7 commented on res/yaml/locale_EN.yml:
```diff
@@ -23,36 +23,36 @@ Headers:
   favoriteEditedOther: "<gold><horse><yellow> is now the favorite horse of <gold><player><yellow>."
   friendAdded: "<gold><player><yellow> has been added to your friends list."
-  friendAddedReverse: "<gold><player><yellow> has added you in his friends list."
+  friendAddedReverse: "<gold><player><yellow> has added you to their friends list."
```
Are you sure about this ? I always wondered if "friends list" was correct but here `<player>` refers to a single player. Thus, 'their' seems to be inappropriate.

jeffadev replied:
Hey, this looks correct in English. We consider "his" to mean a male. By using "their", this means they could either be of Male or Female gender.

Stating the following, for example: "vueu has added you to their friends list.", looks completely proper!

Zedd7 commented on res/yaml/locale_EN.yml:
```diff
@@ -23,36 +23,36 @@ Headers:
   friendRemovedReverse: "<gold><player><yellow> has removed you from his friends list."
   horseAlreadyClaimed: "<yellow>You have already claimed this horse."
-  horseAlreadyOnSale: "<gold><horse><yellow> is already on sale."
+  horseAlreadyOnSale: "<gold><horse><yellow> is already for sale."
```
`horseAlreadyOnSale` should then be changed to `horseAlreadyForSale`

jeffadev replied:
Yes, I did notice this, however I saw no breaking change in the plugin's functionality changing just the chat message. But if you'd like to keep consistency I will gladly go through and change the references in the code too.

Zedd7 replied:
Oh no don't worry about that, this message was for me^^
EDIT : Well, you already figured out ;)

Zedd7 commented on res/yaml/locale_EN.yml:
```diff
@@ -166,7 +166,7 @@ Economy:
 
 Horse informations:
-  health: "<yellow>Health : <gold><amount><yellow>/<gold><max>"
+  health: "<yellow>Health: <gold><amount><yellow>/<gold><max>"
```
I would rather remove the space for all `Horse informations` indexes

Zedd7 reviewed (changes requested):
Thanks again ! Please take a look at this review because I'm not sure about a few things

XIbalba committed:
Merge branch '1.6.8' into 1.6.8

jeffadev committed:
Change horseAlreadyOnSale to horseAlreadyForSale

To fix confusion between on sale and for sale. Typically ON sale means it is at a reduced price. FOR sale means that you are now selling it.

vueu commented:
There, please make sure I did not destroy anything with that commit. :) 👍

jeffadev committed:
Fix rest of the colons on Horse Information

Xibalba commented:
Thank you ! It feels good to know that the EN locale is now correct :)

vueu commented:
Very glad I could help.
